compiler/src/dotty/tools/dotc/typer/ImportInfo.scala

```diff
@@ -28,7 +28,7 @@ object ImportInfo {
  *  @param   selectors     The selector clauses
  *  @param   symNameOpt    Optionally, the name of the import symbol. None for root imports.
  *                         Defined for all explicit imports from ident or select nodes.
- *  @param   importImplied true if this is an implied import
+ *  @param   importImplied true if this is a delegate import
```

Comment:
importImplied

Reply:
Did not have time to fix this yet. Right now only the surface is changed.